pipelines/cflinuxfs4.yml

```diff
@@ -11,390 +11,709 @@ params:
+      - task: create-capi-release-with-rootfs
+        file: buildpacks-ci/tasks/create-capi-release-with-rootfs/task.yml
+        params:
+          STACK: cflinuxfs4
+    - put: cflinuxfs4-rootfs-smoke-test-deployment
+      params:
+        source_file: deployment-source-config/source_file.yml
+        manifest: rootfs-release-artifacts/manifests/manifest.yml
+        releases:
+        - rootfs-release-artifacts/dev_releases/cflinuxfs4/*.tgz
+        stemcells:
+        - gcp-stemcell/*.tgz
+    - task: run-rootfs-smoke-test
+      file: buildpacks-ci/tasks/run-rootfs-smoke-test/task.yml
+      params:
+        ENV_NAME: cflinuxfs4
+    - put: cflinuxfs4-cf-deployment
+      params:
+        source_file: deployment-source-config/source_file.yml
+        manifest: cf-deployment/cf-deployment.yml
+        releases:
+        - rootfs-release-artifacts/dev_releases/cflinuxfs4/*.tgz
+        - capi-release-artifacts/dev_releases/capi/*.tgz
+        stemcells:
+        - gcp-stemcell/*.tgz
+        ops_files:
+        - buildpacks-ci/deployments/operations/no-canaries.yml
+        #! Because currently cflinuxfs3 is the default in cf-deployment
+        - buildpacks-ci/deployments/operations/substitute-with-cflinuxfs4-trusted-certs.yml
+        #! We do not have cflinuxfs4 based buildpacks yet
+        - buildpacks-ci/deployments/operations/do-not-pre-install-buildpacks.yml
+        #! temporary because we need https://github.com/cloudfoundry/garden-runc-release/pull/229
+        - buildpacks-ci/deployments/operations/use-garden-runc-1-20.7.yml
+        - rootfs-release-artifacts/use-dev-release-opsfile.yml
+        - capi-release-artifacts/use-dev-release-opsfile.yml
+        - capi-release-artifacts/use-rootfs-as-default-stack.yml
+        - cf-deployment/operations/use-latest-stemcell.yml
+        - cf-deployment/operations/disable-dynamic-asgs.yml
+        vars:
+          system_domain: cflinuxfs4.buildpacks-gcp.ci.cf-app.com
+    on_failure: #@ failure_alert()
+
+- name: test
+  serial: true
+  serial_groups: [ cflinuxfs4 ]
+  public: true
+  plan:
+  - in_parallel:
+    - get: test-go-buildpack
+    - get: bbl-state
+    - get: buildpacks-ci
+    - get: previous-rootfs-release
+      resource: cflinuxfs4-github-tags
+      passed: [ deploy ]
+    - get: new-cves
+      passed: [ deploy ]
+    - get: stack-s3
+      passed: [ deploy ]
+    - get: version
+      passed: [ deploy ]
+      trigger: true
+    - get: receipt-s3
+      passed: [ deploy ]
+    - get: rootfs
+      resource: cflinuxfs4
+      passed: [ deploy ]
+  - do:
+    - task: get-cf-creds
+      file: buildpacks-ci/tasks/get-cf-creds/task.yml
+      params:
+        ENV_NAME: cflinuxfs4
+    - task: run-test
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: cfbuildpacks/ci
+        inputs:
+        - name: test-go-buildpack
+        - name: cf-admin-password
+        run:
```

Comment:
This is temporary substitute for CATS until we have buildpacks working against cflinuxfs4